src/main/resources/schemas/ont-metadata.schema.json

```diff
@@ -0,0 +1,56 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "http://qbic.life/v1/ont-metadata.schema.json",
+  "title": "Nanopore Instrument Metadata Output",
+  "description": "Describes in which form nanopore metadata is received from the lab.",
+  "type": "object",
+  "properties": {
+    "asic_temp": {
+      "type": "string",
+      "minLength": 1
+    },
+    "device_type": {
+      "type": "string",
+      "minLength": 1
+    },
+    "flow_cell_id": {
+      "type": "string",
+      "minLength": 1
+    },
+    "flow_cell_position": {
+      "type": "string",
+      "minLength": 1
+    },
+    "flow_cell_product_code": {
+      "type": "string",
+      "minLength": 1
+    },
+    "guppy_version":  {
+      "type": "string",
+      "minLength": 1
+    },
+    "hostname": {
+      "type": "string",
+      "minLength": 1
+    },
+    "protocol": {
```

Comment:
Is there a reason, that the "protocol" property doesn't have a "minLength" attribute? Can there be cases where no protocol is needed/specified?

Reply:
well spotted! I will add it in a new PR

Reply:
ah, I defined a pattern:

"SQK-.*(?=:)"

that is why the minLength property is not needed here

Reply:
Ah, that's a good point!